app/src/main/res/layout/download_item.xml

```diff
@@ -18,31 +18,34 @@
     android:adjustViewBounds="true"
     android:minHeight="?android:attr/listPreferredItemHeight"
     android:scaleType="fitCenter"
-    android:src="@mipmap/kiwix_icon" />
+    android:src="@mipmap/kiwix_icon"
+      tools:ignore="ContentDescription,RtlHardcoded" />
```

Comment:
Instead of suppressing missing ContentDescription, i think you have to provide the ContentDescription.